src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -536,4 +539,20 @@
      */
     List<BannerPatternShape> getBannerPatternShapes();
 
+    /**
+     * Retrieves the ItemDictionary for this GameRegistry
```

Comment:
I take the blame. I did not expect the original PR to be merged or used at all, so I didn't perfect it. There are still a few rough spots that need some smoothing and thought. I can fix those parts if needed.